Updated the filter naming in Post Analytics

lenabaidakova opened the pull request:
closes TryGhost/Team#2329
- Replace 'Received' on 'Sent' in member's filter
- Moved links for feedback analytics from chart to table

lenabaidakova committed:
Updated the filter naming in Post Analytics

closes TryGhost/Team#2329
- Replace 'Recived' on 'Sent' in member's filter
- Moved links for feedback analytics from chart to table

moreofmorris commented:
This is looking good so far, I think these changes make sense seeing them in practice. Curious what others will think, too!

I did notice a few bits... If there is only More Like This or only Less Like This available, then we maybe only show that one so people aren't clicking through to no members in the filtered list.
![CleanShot 2022-11-30 at 10 10 38@2x](https://user-images.githubusercontent.com/244516/204768936-bd252738-371c-450c-964b-0100bff1a860.png)

And though "Sent email" shows on the column, when I click around in the filter, for some reason it shows "Received email" again. I can't seem to reliably reproduce this, it seems if I just pick some random emails from the list.
![CleanShot 2022-11-30 at 10 13 40@2x](https://user-images.githubusercontent.com/244516/204769296-13b13297-8f36-434e-a012-444a56727363.png)

Hope that's cool!

lenabaidakova committed:
fixup! Updated the filter naming in Post Analytics

lenabaidakova committed:
fixup! Updated the filter naming in Post Analytics

moreofmorris reviewed (approved):
This is looking great now, thanks @lenabaidakova!

lenabaidakova commented:
> This is looking good so far, I think these changes make sense seeing them in practice. Curious what others will think, too!
> 
> I did notice a few bits... If there is only More Like This or only Less Like This available, then we maybe only show that one so people aren't clicking through to no members in the filtered list. ![CleanShot 2022-11-30 at 10 10 38@2x](https://user-images.githubusercontent.com/244516/204768936-bd252738-371c-450c-964b-0100bff1a860.png)
> 
> And though "Sent email" shows on the column, when I click around in the filter, for some reason it shows "Received email" again. I can't seem to reliably reproduce this, it seems if I just pick some random emails from the list. ![CleanShot 2022-11-30 at 10 13 40@2x](https://user-images.githubusercontent.com/244516/204769296-13b13297-8f36-434e-a012-444a56727363.png)
> 
> Hope that's cool!

@moreofmorris , good catch! 🚀 
- Fixed **More/Less like this** links, now should be visible if any info available
- Fixed column **Received email**

All should be good 😊

lenabaidakova committed:
fixup! Updated the filter naming in Post Analytics
